applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -187,51 +197,63 @@ int le_audio_send(uint8_t const *const data, size_t size)
 	static bool wrn_printed[CONFIG_BT_ISO_MAX_CHAN];
 	struct net_buf *buf;
 
+	/* TODO: This should all be based on number of audio channels */
+	uint8_t *p_data = (uint8_t *)data;
+	size_t num_streams = ARRAY_SIZE(streams);
+	size_t data_size = size/num_streams;
+
+
```

Comment:
Move into for-loop

Reply:
Done.

Reply:
> Same in le_audio_disable(), must check both streams

Not sure about this as the bt_audio_broadcast_source_stop() seems to stop the BIG so calling more than once will error, I think. This may take a little more thought and I should look at this as part of my new task? 
le_audio_disable() is not called currently.

Reply:
Yeah, it would be great if you look at this in your new task